service/src/main/java/org/sunbird/actor/otp/OTPActor.java

```diff
@@ -396,11 +397,13 @@ private void verifyOTPV3(Request request) {
               request.getRequestContext(),
               "OTP_VALIDATION_SUCCESS:OTPActor:verifyOTP: Verified successfully Key = "
                       + OTPUtil.maskId(key, type));
-      int length = 16;
       Map<String, Object> parametersMap = new HashMap<>();
       parametersMap.put(JsonKey.TYPE, type);
       parametersMap.put(JsonKey.KEY, key);
-      String contextToken = generateRandomString(length);
+      Map<String,String> contextDetailsMap =  new HashMap<>();
+      contextDetailsMap.put(JsonKey.CONTEXT_TYPE,(String) otpDetails.get(JsonKey.CONTEXT_TYPE.toLowerCase()));
```

Comment:
toLowerCase should not be used.
Add the conversion for the column name in cassandratablecolumn.properties file

Reply:
1. Added the entries to the cassandratablecolumn.properties